### PR TITLE
LPS-126649| Master

### PR DESCRIPTION
**Pull request by CalumR23**
[LPS-126649](https://issues.liferay.com/browse/LPS-126649)
QA Review Needed
@david-gutierrez-mesa
@Tim-Cao
@liulukiwi

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 1]
```diff
@@ -1761,8 +1761,74 @@ definition {
...
+		}
+
+		task ("Open the Preview modal") {
+			Click(locator1 = "PageEditor#PREVIEW");
+
+			SelectFrame(locator1 = "IFrame#MODAL_BODY");
+		}
+
+		task ("Assert the web content in Preview Modal") {
+			AssertElementPresent(locator1 = "//p[contains(.,'Web Content Title 1')]");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1830]: Should asset the web content is shown in Asset Publisher and WCD and Heading in Preview Modal

**Comment by Tim-Cao** [comment]
ci:stop

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 2]
```diff
@@ -1761,8 +1761,74 @@ definition {
 	test PreviewContentPageViaMappedFragmentWithWCDAndAP {
 		property portal.acceptance = "false";
 
-		// TODO LPS-125933 PreviewContentPageViaMappedFragmentWithWCDAndAP pending implementation
+		task ("Add a widget page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1764]: Unnecessary step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase [hunk 3]
```diff
@@ -1761,8 +1761,74 @@ definition {
 	test PreviewContentPageViaMappedFragmentWithWCDAndAP {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithMapping.testcase:1761]: Rename to PreviewContentPageWithMappedAndInlineContent

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3562
View total diff https://github.com/liferay-echo/liferay-portal/pull/3562/commits/cece0f208e8f68c5e5d11685fea9631b685ad9fc

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**